hollow/src/main/java/com/netflix/hollow/api/client/HollowClientConsumerBridge.java

```
@@ -29,7 +30,21 @@
     
     static HollowConsumer.BlobRetriever consumerBlobRetrieverFor(final HollowBlobRetriever blobRetriever) {
         return new HollowConsumer.BlobRetriever() {
-            
+
+            @Override
+            public HollowConsumer.HeaderBlob retrieveHeaderBlob(long desiredVersion) {
+                final HollowBlobHeader blob = blobRetriever.retrieveHeaderBlob(desiredVersion);
+                if(blob == null)
+                    return null;
+
+                return new HollowConsumer.HeaderBlob(desiredVersion) {
+                    @Override
+                    public InputStream getInputStream() throws IOException {
+                        return null;
```

maybe throw UnsupportedOperationException instead

---

Ack